specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json

```diff
@@ -815,6 +841,28 @@
           "description": "The identity of the workspace",
           "$ref": "#/definitions/ManagedIdentity"
         },
+        "managedVirtualNetworkSettings": {
+          "description": "Managed Virtual Network Settings",
+          "type": "object",
+          "properties": {
+            "preventDataExfiltration": {
```

Comment:
Is there a typo here, should it be preventDataExfilt**e**ration?

Reply:
No, I don't think it is a typo. It should be `data exfiltration`